cumulus_library_data_metrics/us_core_v4/utils.jinja

```diff
@@ -0,0 +1,10 @@
+{# note: this requires a tmp_simplified table #}
+{% macro select_all_fields(fields) %}
+SELECT
+    id,
+    {% for field in fields %}
+    COALESCE({{ field }}, FALSE) AS {{ field }},
+    {% endfor %}
+    COALESCE({{ fields|join(" AND ") }}, FALSE) AS valid
```

Comment:
The difference between this macro and the code that used to be in every template is the COALESCing